CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/),
 and this project adheres to [Semantic Versioning](https://semver.org/).
 
 ## [Unreleased]
+- `Autocomplete` Fixed items not updating when searching, added defaultValue option, added empty state, fixed open state issue
+- `Flyout` Added an option to handle state change
+- `Datepicker` Fixed an issue where the datepicker spams console when it is not loaded in correctly
```

Comment:
Kleine opmerking, maar volgens mij is het gewoon `when it is not loaded correctly`.